### PR TITLE
Add support for excluded directories

### PR DESCRIPTION
**Pull request by akshit-deepsource**
SCATR will now optionally accept a list of directories (absolute or relative to the `cwd`) from which results are excluded. Say for example, an issue was raised in one of the excluded directory. SCATR will ignore matching any files inside the `excluded_dirs`. Same applies for Autofix.

### CODE REVIEW

File: runner/config.go [hunk 1]
```diff
@@ -61,5 +62,14 @@ func ReadConfig(filePath string) (*Config, error) {
 		config.TestAutofix = meta.IsDefined("autofix")
 	}
 
+	for i, dir := range config.ExcludedDirs {
+		normalized, err := normalizeFilePath(dir)
+		if err != nil {
+			return nil, err
+		}
```
Comment (subham-deepsource) [inline runner/config.go:69]: return for any failure? or should we log the error and continue (iterate further)?

**Review by subham-deepsource — CHANGES_REQUESTED** [review]
I left some minor comments. Looks good otherwise.

File: README.md [hunk 2]
```diff
@@ -43,6 +44,13 @@ expects the file paths in the `analysis_results.json` (output from the
 processor) to either be absolute, or to be relative to the `code_path`. Same
 goes for the `files` flag. Only the files in the `code_path` are tested.
 
+### `excluded_dirs`
+
+SCATR optionally also accepts a list of directories (absolute or relative to the
+`cwd`) from which results are excluded. Say for example, an issue was raised
+in one of the excluded directory. SCATR will ignore matching any files inside
+the `excluded_dirs`. Same applies for Autofix.
```
[the remark this answers is not in the document]
Comment (akshit-deepsource) [reply README.md:52]: I use `cwd` because that is the CLI flag for scatr.

File: runner/config.go (hunk 1 above)
Comment (akshit-deepsource) [reply runner/config.go:69]: Returning because if the configuration parsing fails, analysis won't be able to continue further.

Comment (subham-deepsource) [reply runner/config.go:69]: Ok

**Review by subham-deepsource — APPROVED** [review]

**Pull request merged**